supplementary/alica_ros1/alica_tests/src/test/test_legacy_conditions.cpp

```diff
@@ -1,79 +1,44 @@
-#include "test_alica.h"
-
-#include <alica/test/CounterClass.h>
-#include <alica_tests/Behaviour/Attack.h>
-#include <alica_tests/Behaviour/MidFieldStandard.h>
-#include <alica_tests/LegacyTransitionConditionCreator.h>
-
+#include <alica/test/TestContext.h>
 #include <alica/test/Util.h>
-#include <engine/AlicaClock.h>
-#include <engine/AlicaContext.h>
-#include <engine/AlicaEngine.h>
-#include <engine/Assignment.h>
-#include <engine/BasicBehaviour.h>
-#include <engine/DefaultUtilityFunction.h>
-#include <engine/IAlicaCommunication.h>
-#include <engine/PlanBase.h>
-#include <engine/PlanRepository.h>
-#include <engine/TeamObserver.h>
-#include <engine/model/Behaviour.h>
-#include <engine/model/Plan.h>
-#include <engine/model/RuntimeCondition.h>
-#include <engine/model/State.h>
-
+#include <alica_tests/Behaviour/Attack.h>
+#include <alica_tests/SimpleTestPlan1412252439925.h>
+#include <alica_tests/TestFixture.h>
 #include <gtest/gtest.h>
 
-namespace alica
-{
-namespace
-{
-
-class AlicaLegacyConditions : public AlicaLegacyConditionsFixture
+namespace alica::test
 {
-protected:
-    const char* getRoleSetName() const override { return "Roleset"; }
-    const char* getMasterPlanName() const override { return "SimpleTestPlan"; }
-    bool stepEngine() const override { return false; }
-};
 
-TEST_F(AlicaLegacyConditions, simplePlanTestWithLegacyConditions)
+TEST_F(TestFixture, simplePlanTestWithLegacyConditions)
```

Comment:
Need to use the `LegacyTransitionConditionCreator` since the test name says `WithLegacyConditions`.
Further, I think this test can be completely redesigned, because I don't know what `SimplePlanTest` means or why the test has so many complex counter checks.